tools/generate_dalite_passport.py

```diff
@@ -0,0 +1,26 @@
+"""Utility that allows to generate passports encoded for this xblock."""
+import argparse
+from dalite_xblock.passport_utils import DaliteLtiPassport, prepare_passport
+
+
+def main():
+    """Entrypoint for this script."""
+    parser = argparse.ArgumentParser(description='Create dalite passport')
+    parser.add_argument('--passport-name', help='Name of this passport', required=True)
```

Comment:
This argument is referred to as `passport-id` in the README.  I don't mind which name is used, but could they be made consistent?